config/_default/menus/menus.en.yaml

```diff
@@ -2535,6 +2535,10 @@ main:
     url: synthetics/api_tests/icmp_tests
     parent: api_tests
     weight: 107
+  - name: gRPC Healthchecks
```

Comment:
I see GRPC and gRPC in the gRPC documentation (https://github.com/grpc/grpc/blob/master/doc/health-checking.md), so for consistency purposes, I recommend title case for the header and sentence case for the body text—what are your thoughts?

```suggestion
  - name: GRPC
```